S/SHTns/build_tarballs.jl

```diff
@@ -5,16 +5,17 @@ using BinaryBuilder, Pkg
 include(joinpath(@__DIR__, "..", "..", "platforms", "microarchitectures.jl"))
 
 name = "SHTns"
-version = v"3.5.2"
+version = v"3.6.5"
```

Comment:
Title says 3.6.6 here it is 3.6.5 ?

Reply:
Thanks guys, the fix was embarrassingly easy.
I blindly tried 3.6.5 hoping it might not fail. Not that the build is fixed I will upgrade to 3.6.6 .

Reply:
No worries, I am sure we've all been there at some point or another (and at least in my case, I am sadly confident to claim I'll be there again ;-).

Thanks for sticking through it!